qemu/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM cusdeb/alpine3.7:amd64
+FROM amd64/alpine:3.13
```

Comment:
* `amd64` is used by default. I suggest we will keep our old convention like `FROM alpine:3.13`.
* Don't forget to reflect the change it README:
  ```
  <a href="https://hub.docker.com/_/alpine">Official Docker image</a> based on <a href="https://alpinelinux.org/posts/Alpine-3.13.0-released.html">Alpine 3.13</a>
  ```